storage/storage_bench.go

```diff
@@ -416,7 +416,7 @@ func AnnounceLeecher1kInfohash(b *testing.B, ps PeerStore) {
 // AnnounceSeeder can run in parallel.
 func AnnounceSeeder(b *testing.B, ps PeerStore) {
 	runBenchmark(b, ps, true, putPeers, func(i int, ps PeerStore, bd *benchData) error {
-		_, err := ps.AnnouncePeers(bd.infohashes[0], true, 50, bd.peers[0])
+		_, err := ps.AnnouncePeers(bd.infohashes[0], false, 50, bd.peers[0], nil)
```

Comment:
And this needs `true` as the second parameter, otherwise it tests AnnounceLeecher.